base_report_auto_create_qweb/models/report_xml.py

```diff
@@ -0,0 +1,95 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import models, api, exceptions, _
+
+
+class IrActionsReport(models.Model):
+    _inherit = 'ir.actions.report.xml'
+
+    def _prepare_qweb_view_data(self, qweb_name, arch):
+        return {
+            'name': qweb_name,
+            'mode': 'primary',
+            'type': 'qweb',
+            'arch': arch,
+        }
+
+    def _prepare_model_data_data(self, qweb_name, module, qweb_view):
+        return {
+            'module': module,
+            'name': qweb_name,
+            'res_id': qweb_view.id,
+            'model': 'ir.ui.view',
+        }
+
+    def _prepare_value_view_data(self, name, model):
+        return {
+            'name': name,
+            'model': model,
+            'key2': 'client_print_multi',
+            'value_unpickle': 'ir.actions.report.xml,%s' % self.id,
+        }
+
+    def _create_qweb(self, name, qweb_name, module, model, arch):
+        qweb_view_data = self._prepare_qweb_view_data(qweb_name, arch)
+        qweb_view = self.env['ir.ui.view'].create(qweb_view_data)
+        model_data_data = self._prepare_model_data_data(
+            qweb_name, module, qweb_view)
+        self.env['ir.model.data'].create(model_data_data)
+        value_view_data = self._prepare_value_view_data(
+            name, model)
+        self.env['ir.values'].create(value_view_data)
```

Comment:
A client reported this issue
![image](https://cloud.githubusercontent.com/assets/2323616/9248912/640982cc-41c0-11e5-8f2e-4f84ea268790.png)

I haven't tested but I think it will be fixed by putting `.sudo()` in this line